config/environments/production.rb

```diff
@@ -67,7 +67,7 @@
   # Ignore bad email addresses and do not raise email delivery errors.
   # Set this to true and configure the email server for immediate delivery to raise delivery errors.
   # config.action_mailer.raise_delivery_errors = false
-
+  config.action_mailer.default_url_options = { host: 'localhost', port: 3000 }
```

Comment:
you should set this one on the development.rb file. Adding this to the production env configuration would mean that this is the settings used on the production server.